install/src/main/resources/aspects.bzl

```diff
@@ -488,6 +488,10 @@ def _bsp_target_info_aspect_impl(target, ctx):
     scala_target_info = extract_scala_info(target, ctx, output_groups)
     java_toolchain_info, java_toolchain_info_exported = extract_java_toolchain(target, ctx, dep_targets)
     java_runtime_info, java_runtime_info_exported = extract_java_runtime(target, ctx, dep_targets)
+    cpp_target_info = extract_cpp_target_info(target, ctx)
+
+    # TODO retrieve cpp compiler info and use it
+    cpp_compiler_info = _fetch_cpp_compiler(target, ctx)
```

Comment:
u can remove this call and `_fetch_cpp_compiler` method as well

Reply:
but still leave this #TODO (to somehow remember that issue with cpp compiler is not resolved ) ?